### PR TITLE
Add sipgate-softphone

### PR DESCRIPTION
**Pull request by vknabel**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

I copied the download URL from the [official download website sipgateteam.de/softphone/download](https://www.sipgateteam.de/softphone/download).
The current version is 1.9.12. Though there is no version number within the download url.

### CODE REVIEW

File: Casks/sipgate-softphone.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sipgate-softphone' do
+  version :latest
+  sha256 :no_check
+
+  # sipgate-desktop-app.s3.eu-central-1.amazonaws.com was verified as official when first introduced to the cask
+  url 'https://sipgate-desktop-app.s3.eu-central-1.amazonaws.com/sipgate-softphone.dmg'
```
Comment (miccal) [inline Casks/sipgate-softphone.rb:6]: Please add the following `appcast`:
```
appcast 'https://sipgate-desktop-app-dev.s3.amazonaws.com/latest-mac.yml'
```

File: Casks/sipgate-softphone.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'sipgate-softphone' do
+  version :latest
+  sha256 :no_check
```
Comment (miccal) [inline Casks/sipgate-softphone.rb:3]: Please change these to:
```
version '1.9.12'
sha256 'fa938c5475cfcec00dce8778ffa1e2048922a81f4e4f11d053581b9dd3b742f6'
```

File: Casks/sipgate-softphone.rb (hunk 1 above)
Comment (vknabel) [reply Casks/sipgate-softphone.rb:6]: @miccal thanks for the suggestion.
Just curios: how did you find out this file exists?

Comment (miccal) [reply Casks/sipgate-softphone.rb:6]: Please see the [`appcast` documentation](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Pull request merged**